### PR TITLE
[Jetnews] Handle Window Insets

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

**Comment by chrisbanes** [comment]
I'll hold off reviewing this for now as it's marked as a draft. @JolandaVerhoef let me know if you'd like me to take an early pass through.

**Comment by JolandaVerhoef** [comment]
@chrisbanes It's ready now! It was working on top of the navigation CL, so couldn't be merged yet. But now that one has been merged, so this one is ready to go!

File: JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt [hunk 1]
```diff
@@ -19,21 +19,32 @@ package com.example.jetnews.ui
...
 fun JetnewsApp(
     appContainer: AppContainer
 ) {
     JetnewsTheme {
+        ProvideWindowInsets {
+            val systemUiController = rememberSystemUiController()
+            SideEffect {
+                systemUiController.setSystemBarsColor(Color.Transparent, darkIcons = false)
+            }
+        }
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt:46]: All of the app content should be in the `ProvideWindowInsets` (it sets a composition local)

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt [hunk 2]
```diff
@@ -156,7 +154,12 @@ fun ArticleScreen(
                 onToggleFavorite = onToggleFavorite
             )
         }
-    )
+    ) { innerPadding ->
+        val modifier = Modifier
```
Comment (chrisbanes) [inline JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt:158]: Nit: inline the modifier. Also, needs a comment to explain this

File: JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt (hunk 1 above)
Comment (JolandaVerhoef) [reply JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt]: Done!

File: JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt (hunk 2 above)
Comment (JolandaVerhoef) [reply JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt]: Done!

**Review by chrisbanes — APPROVED** [review]
LGTM, nice work 👍

**Pull request merged**